src/sections/2021/section-5.js

```diff
@@ -144,9 +144,9 @@ const Section5 = () => (
         </svg>
 
         <h3>Design systems are intended to cross complex boundaries, but realizing the full value of your investment is tricky work.</h3>
-        <p>Design systems are intended to cross complex boundaries, but realizing the full value of your investment is tricky work.</p>
+        <p>What do you need to support your design system?</p>
```

Comment:
I did not see this in the content edits doc. Was this from a separate conversation?

Reply:
This is how it is written in the Content Edits Doc. Should I remove this line?

<img width="675" alt="Screen Shot 2021-07-08 at 9 34 55 AM" src="https://user-images.githubusercontent.com/24901036/124930963-c3bb7a00-dfcf-11eb-867e-b24bc0aece81.png">

Reply:
Ah. That’s in the [main doc](https://docs.google.com/document/d/1GZoY2jo4G8bYp9qW1ja8qxHgwiza4zO73El6Zsey2lk/edit#), not in the [edit doc](https://docs.google.com/document/d/1AvTqkMunkK0f1I-iCD6O0ZK9WNKA8gAp4QNt8lPv9yU/edit). That needs to be verified with Kaitlyn.